dao/src/main/kotlin/repositories/DaoOrtRunRepository.kt

```diff
@@ -63,7 +64,15 @@ class DaoOrtRunRepository(private val db: Database) : OrtRunRepository {
         issues: Collection<Issue>,
         traceId: String?
     ): OrtRun = db.blockingQuery {
-        val nextIndex = (listForRepository(repositoryId).data.maxByOrNull { it.index }?.index ?: 0) + 1
+        val lastIndex = OrtRunsTable
+            .select(OrtRunsTable.index)
```

Comment:
An option could be to use the  `max` function instead of the combination of `order by` and `limit`. According to [this article](https://medium.com/@hamida.meknassi/min-max-vs-order-by-limit-performance-dfb6420701ee), this should provide better performance.